sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/keyvault_secret.hpp

```diff
@@ -77,7 +79,131 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
       }
     }
 
+    KeyVaultSecret(_detail::Models::SecretBundle const& secret)
```

Comment:
Similar comment - can this be moved to a .cpp file to hide the implemetnation details from the published headers? Then you can declare without defining the SecretBundle.

Reply:
done as requested :)